APE8.rst

```diff
@@ -0,0 +1,94 @@
+Astropy Community Code of Conduct
+---------------------------------
+
+authors: Kelle Cruz, Alex Hagen, Thomas Robitaille, Erik Tollerud, Alexa Villaume
+
+date-created: 2015 May 4
+
+date-last-revised: 2015 May 4
+
+type: Informational
+
+status: Discussion
+
+
+Abstract
+--------
+
+This document and APE grew out of discussion at the April 2015 Lorentz Center
+meeting on Python in Astronomy, prompted both by meetings involving a
+substantial group from this community, as well as the addition of commenting
+capabilities to the Astropy Tutorials page.  We believe that our community has
+developed to the point that we should have a code of conduct specific to the
+Astropy project (and other projects of similar mind).  This APE provides the
+text of just such a code of conduct, which will be adopted by the project to
+provide guidance to the community. The code of conduct will be licensed using
+the Creative Commons Attribution license, allowing other projects to use this
+code or adapt it to their needs.
+
+
+Detailed description
+--------------------
+
+The text of the code of conduct is shown here:
+
+Astropy Community Code of Conduct
+=================================
+
+The community of participants in open source Astronomy projects is made up of members from around the globe with a diverse set of skills, personalities, and experiences. It is through these differences that our community experiences success and continued growth. We expect everyone in our community to follow these guidelines when interacting with others both inside and outside of our community. Our goal is to keep ours a positive, inclusive, successful, and growing community.
+
+As members of the community,
+
+* We pledge to treat all people with respect and provide a harassment- and bullying-free environment, regardless of sex, sexual orientation and/or gender identity, disability, physical appearance, body size, race, nationality, ethnicity, and religion. In particular, sexual language and imagery, sexist, racist,  or exclusionary jokes are not appropriate.
```

Comment:
Extra space between "racist, or".    I wonder if it would make sense to say "otherwise exclusionary" given that the previously listed items are already exclusionary by their nature.  On the other hand maybe that's implicit and I'm being characteristically too wordy.